GVFS/GVFS.Installer.Mac/scripts/Distribution_GVFS.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
+<installer-gui-script minSpecVersion="1">
+    <pkg-ref id="com.vfsforgit.pkg"/>
+    <options customize="never" allow-external-scripts="true"/>
+    <choices-outline>
+        <line choice="default">
+            <line choice="com.vfsforgit.pkg"/>
+        </line>
+    </choices-outline>
+    <choice id="default"/>
+    <choice id="com.vfsforgit.pkg" visible="false">
+        <pkg-ref id="com.vfsforgit.pkg"/>
+    </choice>
+    <pkg-ref id="com.vfsforgit.pkg" version="1.2.3" onConclusion="none">VFSFORGITPACKAGEFILENAME</pkg-ref>
```

Comment:
`version="1.2.3"` -> Where did this version number come from?

Reply:
that was just a placeholder. Updated the build script to update `Distribution.xml` with actual values computed during build time.